topic-10-challenges/unit-1/book-b/03.03.md

```diff
@@ -0,0 +1,42 @@
+# How
+
+To create the `index.html` file you should build on the knowledge that was gained in the last challenge.
+
+The first part of the command should create the header information down to the start of the list.
+```html 
+<h1>Proffessor Index</h1>
+<ul>
+``` 
+
+Then we should loop over the professors to and add them to the list. 
+When the redirect `>>` you can append to a file, while the `>` will replace the exist file.
+
+The elements in the list should follow the format given,
+```html
+<li><a href=./FILENAME>PROFESSOR NAME</a></li>
+```
+Or
+
+```html
+<li><a href=./EvaHill.html>Prof. John Doe</a></li>
+``` 
+
+After the all the professors have being added to the list you must close the list.
+The list can be closed by appending `</ul>` to the end of the file.
+
+```html
+</ul>
+```
+
+The final file should look like:
+
+```html 
+<h1>Proffessor Index</h1>
+<ul>
+<li><a href=./EvaHill.html>Prof. John Doe</a></li>
+<li><a href=./MichaelHill.html>Prof. Jane Smith</a></li>
+</ul>
+``` 
+
+Some of the concepts that would be good to know when trying to achieve this are, loops, appending and the `&&` which allows running commands after the current has completed.
+The command may take the following format, `Adding header && looping over professers && adding the footer`.
```

Comment:
I think the pseudocode for chaining commands may be confusing for them but i understand the need for it. Not immediately sure how I'd improve this, maybe:

(Header Command) && (Looping Command) && (Footer Command)

Reply:
Ya this is a harder one to show. I wrap them in `<>` over the `()` as they have a meaning in bash also.